jackson-xml/build.gradle.kts

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id("io.micronaut.build.internal.jackson-xml-module")
+}
+
+dependencies {
+    api(libs.jackson.dataformat.xml)
+    api(mn.micronaut.inject)
+    api(mn.micronaut.http)
+    implementation(mn.micronaut.jackson.databind)
```

Comment:
I added a dependency to `micronaut.jackson.databind` because has a `JacksonMediaTypeCodec` `@Singleton` `XmlMediaTypeCodec`.